frontend/src/containers/allocations.js

```diff
@@ -33,7 +49,7 @@ class Allocations extends Component {
                                                 <td><NomadLink allocId={allocation.ID} short="true"/></td>
                                                 <td><NomadLink jobId={allocation.JobID} short="true"/></td>
                                                 <td>{allocation.ClientStatus}</td>
-                                                <td>{allocation.DesiredStatus}</td>
+                                                <td>{this.renderDesiredStatus(allocation)}</td>
```

Comment:
yeah, experimented with a small fa-icon next to it, but couldn't agree on something

considering a dotted underline might be a good solution now